TheMovie/TheMovie/Scenes/Login/MainScreen/View/MainViewController.swift

```diff
@@ -2,6 +2,8 @@ import UIKit
 
 class MainViewController: UIViewController {
     
+    private let viewModel = MainViewModel()
```

Comment:
vamo começar a trabalhar com injeção de dependências?
Injeta isso daí na controller através de uma factory, ela vai ser responsável por administrar toda essa parte, e vc n vai construir a classe pura aqui, vamos utilizar protocolos tb

Reply:
To em duvida em como faria isso